oam/install_scripts/run.sh

```diff
@@ -3,20 +3,23 @@
 # $Id$
 #
 
-usage="usage: run.sh [-vh] [-s sleep] [-t tries] executable"
+usage="usage: run.sh [-vh] [-s sleep] [-t tries] [-l logDir] executable"
```

Comment:
Is this supposed to be 'lockDir'?  Looks like all we do with it is test for StopColumnstore?  If that's the case, and this is an important check, we should get the path from a reliable source rather than as a cmdline option.